### PR TITLE
Pro 4247 create sprod env

### PR DESCRIPTION
**Pull request by Bishwin**
https://tools.hmcts.net/jira/browse/PRO-4247

### CODE REVIEW

File: infrastructure/sprod.tfvars [hunk 1]
```diff
@@ -1,50 +1,44 @@
...
-asp_rg = "probate-sprod"
-feature_toggles_api_url = "http://rpe-feature-toggle-api-sprod.service.core-compute-sprod.internal"
\ No newline at end of file
+reform_envirionment_for_test = "saat"
+
+vault_section = "test"
+
+probate_frontend_hostname = "probate-frontend-sandbox-sprod.service.core-compute-sprod.internal"
+outbound_proxy = ""
+probate_redis_url = "betaDevBprobatecache01.reform.hmcts.net"
```
Comment (reddy1praveen) [inline infrastructure/sprod.tfvars]: This is also not required

File: infrastructure/sprod.tfvars [hunk 2]
```diff
@@ -1,50 +1,44 @@
...
-
 probate_frontend_use_redis = "true"
 
-reform_envirionment_for_test = "saat"
+probate_google_track_id = "UA-93598808-1"
 
-probate_business_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4101/validate"
+probate_private_beta_auth_url = "https://idam-test.dev.ccidam.reform.hmcts.net/login"
 
+probate_business_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4101/validate"
```
Comment (reddy1praveen) [inline infrastructure/sprod.tfvars]: probate_business_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4101/validate"
probate_persistence_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4103/formdata"
probate_submit_service_url = "http://betaDevBprobateApp01.reform.hmcts.net:4102/submit"

Comment (reddy1praveen) [reply infrastructure/sprod.tfvars]: remove this

Comment (Bishwin) [reply infrastructure/sprod.tfvars]: removed

File: infrastructure/sprod.tfvars (hunk 1 above)
Comment (Bishwin) [reply infrastructure/sprod.tfvars]: removed

**Pull request merged**